IT/pom.xml

```diff
@@ -5,7 +5,7 @@
     <parent>
         <groupId>com.microsoft.gctoolkit</groupId>
         <artifactId>gctoolkit</artifactId>
-        <version>3.0.3-SNAPSHOT</version>
+        <version>3.0.3</version>
```

Comment:
Doesn't our release task make the non SNAPSHOT version number?

Reply:
I'll revert

Reply:
revised to using the revision mechanism (https://maven.apache.org/maven-ci-friendly.html)